src/components/UsersFilters.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {FormControl, Grid, InputLabel, MenuItem, Select} from "@material-ui/core";
+
+import PropTypes from "prop-types";
+import useStyles from './usersViewStyles';
+
+function UsersFilters({ selectedNationality, selectedGender, handleChangeNationality, handleChangeGender }) {
```

Comment:
Fajnie, że filtry są w oddzielnym komponencie. Gdyby było więcej opcji w selectach to można by zrobić consty z listami do mapowania